### ISSUE TITLE
Question about the firstday sql

### ISSUE THREAD

**Issue by ljz756245026** [MIT-LCP/mimic-code#1565]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

**Comment by mlbernauer** [comment]
I suspect the 6hr buffer is used to ensure that events occurring in the ICU prior to official ICU admission are captured. I.e. to address any sort of administrative lag between when the patient was physically moved to the ICU and when the admission was documented.

**Comment by alistairewj** [comment]
Yes, absolutely that is your choice. I look 6 hours before ICU admission as the admission time is administrative. For labs, there may be ED labs occurring just before ICU admission, and I wanted to include those in the query. For vital signs, they may have been connected to the monitor just before admission, and I wanted to capture that as well. For consistency I chose 6 hours across all the queries. You could do exactly intime, but I'd just caution that none of those in times are exactly when the patient physically entered the ICU. You can compare it to the time of the first heart rate measurement to see what I mean.

**Comment by ljz756245026** [comment]
> 

Thank you for your quick reply.
I want to know that whether there is a paper or some medical sources to support the use of 6 hours.

**Comment by alistairewj** [comment]
You'd be lucky to get that kind of detail in a paper! One of the main contributors to challenges in reproducibility.

Best approach would be to create a histogram of the time between `intime` and `charttime` for various common measurements you're interested in, and see what comes out. There isn't much justification for the exact 6 hour number, it was just intuition. Could very well be improved!

**Comment by ljz756245026** [comment]
> You'd be lucky to get that kind of detail in a paper! One of the main contributors to challenges in reproducibility.
> 
> Best approach would be to create a histogram of the time between `intime` and `charttime` for various common measurements you're interested in, and see what comes out. There isn't much justification for the exact 6 hour number, it was just intuition. Could very well be improved!

I will appreciate it if you could tell me which paper contains such detail. Thank you!

**Comment by alistairewj** [comment]
Sorry I wasn't clear. I don't know of any paper which justifies the use of 6 hours.
